networking_cisco/plugins/cisco/cfg_agent/device_drivers/asr1k/aci_asr1k_routing_driver.py

```diff
@@ -82,6 +85,24 @@ def internal_network_added(self, ri, port):
                 else:
                     self._add_tenant_net_route(ri, port)
 
+    def external_gateway_removed(self, ri, ext_gw_port):
```

Comment:
most of the code is the same as parent's class except for last 2 newly added lines, let's refactor then re-use the code as much as possible then.